bin/gobuild.sh

```diff
@@ -31,8 +31,14 @@ BUILDPATH=${3:?"path to build"}
 
 set -e
 
+ARCH=`uname -m`
+
 GOOS=${GOOS:-linux}
 GOARCH=${GOARCH:-amd64}
+if [ "$ARCH" == "ppc64le" ]; then
+    GOARCH=$ARCH
```

Comment:
If I read this correctly, then someone running on a PPC system is forced to compile for PPC (i.e., cross-compiling isn't possible).  I'm thinking it'd be more appropriate to have something like the following (though there's probably a more concise way to get the same result):

<pre>
if [ "$ARCH" == "ppc64le" ]; then
    DEFAULT_ARCH=$ARCH
else
    DEFAULT_ARCH=amd64
endif

GOARCH=${GOARCH:-${DEFAULT_ARCH}}
</pre>

Reply:
This suggestion makes sense. Have added the required changes.